app/assets/stylesheets/passwords/components/_new_desktop.scss

```diff
@@ -0,0 +1,39 @@
+.password-container {
+  margin-bottom: 20vh;
+
+  .password-form {
+    width: 400px;
+
+    .field {
+      .form-control {
+        width: 380px;
+      }
+    }
+
+    .actions {
+      .btn-primary {
+        width: 380px;
+      }
+    }
+
+    .links {
+      left: -155px;
+
+      .btn-sign-in {
+        margin-bottom: 30px;
+
+        .sign-up {
+          padding: 10px;
+        }
```

Comment:
I recommend you to avoid more than 4 levels deep when you are nesting in scss:
```
The Inception Rule: don’t go more than four levels deep. Some say it’s possible to develop deepfakes following these instructions. 

This basically means that you shouldn’t be too specific or mimicking the DOM at any point. If you find yourself more than four levels deep, that’s a red flag. Of course there are times when you’ll be forced to go there, but it’s not something you should be doing too much.
```
https://thesassway.com/beginner-the-inception-rule/